server/controllers/authController.js

```diff
@@ -6,21 +6,39 @@ const usersModel = new UsersModel();
 
 const authController = {
   /**
-   * User sign up handler
+   * User(client) sign up handler
    * @param {Object} req - server request object
    * @param {Object} res - server response object
    * @returns {JSON} res - custom server response
    */
   async signUp(req, res) {
     try {
       const user = await usersModel.create(req.body);
+      user.password = undefined;
       const token = authHelper.generateToken(user);
       controllerResponse.successResponse(res, 201, user, token);
```

Comment:
Now, that you made password undefined, how does the responses look

Reply:
The response looks better than before. I can add more screenshots.